### DISCUSSION TITLE
Can i have combox inpup and popover in the same div

### DISCUSSION THREAD

**Discussion by johnsonthedev** [ariakit/ariakit#3226] [Questions] [answered]
hiii, 

I am at the airport and waiting for my flight :)

I played a little bit with the comobox and was wondering if I could style it similair to the google search. Here is a snipped to show the idea:

https://play.tailwindcss.com/v6VKPImZlG

Do you think something like this would be possible. 

I think not as the pop-up is floating.

Just wanted to ask.

Thx John

**Comment by diegohaz** [comment, marked as answer]
Sure, it's a bit of a hack at the moment, but you can pass an empty function to the [`updatePosition`](https://ariakit.org/reference/combobox-popover#updateposition) prop and modify the popover wrapper styles using [`wrapperProps`](https://ariakit.org/reference/combobox-popover#wrapperprops):

```jsx
<ComboboxPopover
  updatePosition={() => {}}
  wrapperProps={{ style: { position: "relative" } }}
>
```
